src/charm.py

```diff
@@ -534,20 +534,19 @@ def _execute_migrations(self) -> None:
         if not self._are_relations_ready() or not container.can_connect():
             logger.info("Not ready to execute migrations")
             return
-        if self.model.unit.is_leader():
-            env_settings = self._create_discourse_environment_settings()
-            self.model.unit.status = MaintenanceStatus("Executing migrations")
-            try:
-                migration_process: ExecProcess = container.exec(
-                    [f"{DISCOURSE_PATH}/bin/bundle", "exec", "rake", "--trace", "db:migrate"],
-                    environment=env_settings,
-                    working_dir=DISCOURSE_PATH,
-                    user=CONTAINER_APP_USERNAME,
-                )
-                migration_process.wait_output()
-            except ExecError as cmd_err:
-                logger.exception("Executing migrations failed with code %d.", cmd_err.exit_code)
-                raise
```

Comment:
I think it might just be worth a code comment explaining that it's okay to run this on every unit as we've confirmed the process is idempotent, with a reference if possible.

Reply:
Got it, I updated the code to add a comment explaining this